versions.yaml

```diff
@@ -16,13 +16,13 @@
 
 _variables:
   wordpress: &wordpress
-    version: '5.3.2'
```

Comment:
5.3 is used in k8s applications, please leave both 5.3 and 5.7 versions

Reply:
done